### ISSUE TITLE
support vanilla models.jag

### ISSUE THREAD

**Issue by alarixnia** [2003scape/rsc-c#142]
to do this, we need to fix some problems in the models that the gl renderer doesn't like in the code. For example removing excess polygons that are far away from the origin

**Comment by misterhat** [comment]
yup. alternatively we can re-do how mouse picking works by using a colours like emscripten's does for landscape. each object would get a unique colour associated with its position and rendered in another context. for now it'd be easier to just cut off the stray faces and continue with bounding box picking.

bad models include:
* wallclockface
* market
